CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.0] - 2021-09-09
+
+- Loading rules form multiple ZooKeeper nodes in Siembol alerting
+- Fixing adding tags in Siembol correlation alerts
+- various minor fixes in Sigma rule importer in Siembol UI
+- various minor fixes in Siembol UI - uppercase searches for tags, tags in deployment dialog respect json schema required property
```

Comment:
for tags in deployment dialog maybe just:
tags in deployment dialog validated before release